bootstrap/sql/postgresql/drop_tables.sql

```diff
@@ -1,44 +0,0 @@
-DROP TABLE IF EXISTS file CASCADE;
```

Comment:
I assume flyway is going to drop the tables. Does it take care of dropping the child tables first?. I think we had some issues in the past when the order of "drop" was not correct.